source/expressions.tex

```diff
@@ -7750,7 +7750,7 @@
 it is applied to
 \begin{itemize}
   \item
-  a glvalue of type \cv \tcode{std::nullptr_t},
+  a glvalue of type \cv~\tcode{std::nullptr_t},
```

Comment:
The usual spelling seems to be `\cv{}~...`, could you please amend accordingly?

Reply:
done